packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.tsx

```diff
@@ -17,11 +16,10 @@ export class PrimaryButton extends BaseComponent<IButtonProps, {}> {
     const { theme, styles } = this.props;
 
     return (
-      <BaseButton
+      <DefaultButton
         { ...this.props }
-        variantClassName='ms-Button--primary'
```

Comment:
I think that we should try to keep this separate from DefaultButton, or make DefaultButton more flexible so it can take in varient names or something.